acac_backend/src/main/models.py

```diff
@@ -152,10 +152,11 @@ class Senate(models.Model):
     cover = VersatileImageField('Cover', upload_to='society_%Y', help_text="Upload high quality picture")
     skin = models.CharField(max_length=32, choices=SKIN_CHOICES, blank=True, default='mdb-skin',
                             help_text="Choose a skin while displaying senate page.")
-    members = models.ManyToManyField(UserProfile, through='SenateMembership', through_fields=('senate', 'userprofile'))
-    coordinator_student = models.ForeignKey(Faculty, blank=True, null=True, default=None,
-                                            on_delete=models.SET_NULL)
-    report_link = models.URLField(help_text='Add a drive link of annual report to show on board page', null=True, blank=True)
+    gen_secy_senate = models.ForeignKey(UserProfile, related_name='gen_secy_senate', limit_choices_to={'user__is_staff': True},
```

Comment:
You forgot to remove `coordinator_student` and add `gen_secy_senate` and `gen_secy_acac` in `senatefixture.py` file